src/encoded/types/characterization.py

```diff
@@ -24,6 +36,32 @@ class Characterization(ItemWithAttachment, Item):
         'documents',
     ]
     set_status_down = []
+    STATUS_ACL = {
+        'in progress': ALLOW_REVIEWER_EDIT,
+        'released': ALLOW_CURRENT,
+        'not submitted for review by lab': ALLOW_CURRENT,
+        'deleted': DELETED,
+    }
+
+    def __ac_local_roles__(self):
+        roles = {}
```

Comment:
This should be something like `roles = super().__acl_local_roles__()` to prevent code duplication. Then add to the roles dictionary before returning it.